### PR TITLE
(2773) Report uploading emails

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- Report approved emails sent to BEIS users include a reminder that the report is going to be uploaded
- Send notification email to report approver if report upload fails

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by CristinaRO** [comment]
[Report uploading: customise the failure notification email](https://trello.com/c/kJ3qwXST/2773-report-uploading-customise-the-failure-notification-email)

File: app/jobs/report_export_uploader_job.rb
Comment (Gweaton) [inline app/jobs/report_export_uploader_job.rb:14]: I've asked about this on Slack - the BEIS team would rather we only sent this to the report approver to prevent wider confusion. We'll probably want to update the wording in the approval confirmation email to say "The person who approved the report will receive a notification if the upload fails" (or words to that effect)

**Review by Gweaton — APPROVED** [review]
Looks good - just one thing re: only sending notification to one person. I'll approve for now though so it doesn't block you merging later on.

File: app/views/report_mailer/upload_failed.text.erb
Comment (yndajas) [inline app/views/report_mailer/upload_failed.text.erb:3]: The first `report` in `report CSV for the report` might be a but redundant

**Pull request merged**
